### PR TITLE
Show 100 of total if > 100

### PR DESCRIPTION
**Pull request by alansutherland**

### CODE REVIEW

File: pages/set/_.vue [hunk 1]
```diff
@@ -97,7 +97,11 @@
       <b-row>
         <b-col>
           <h2 class="related-heading text-uppercase">
-            {{ $tc('items.itemCount', itemCount, { count: itemCount }) }}
+            {{
+              itemCount > 100 ?
+              $tc('items.itemOf', itemCount, { count: itemCount }) :
+              $tc('items.itemCount', itemCount, { count: itemCount })
```
Comment (lbiedinger) [inline pages/set/_.vue:103]: these two lines need to be indented one more level as they're par t of the previous statement.
Although I think having to split ternary operators onto multiple lines like this is also not great. Let's move the whole thing into a computed property and have the logic there as follows:

```
 computed: {
   ...
   displayItemCount() {
      const label = this.set.total > 100 ? 'items.itemOf' : 'items.itemCount';
      return this.$tc(label, this.this.set.total) :
   }
```
Which can replace the 'itemCount' property as that was only used here anyway.

Also note the change of what's passed to the $tc function, I don't know why count was being passed twice, but I suspect the intention was once for pluralization and once for the value, however once should suffice. I tried it and it seems to work for 0, 1, 2...100 & 101 items, but worth double checking.

**Review by lbiedinger — CHANGES_REQUESTED** [review]

File: pages/set/_.vue (hunk 1 above)
Comment (alansutherland) [reply pages/set/_.vue]: Yes, this approach makes more sense than the ternary operator. I'm not sure why the count was being passed twice before but as you say it's not needed. I've tested multiple different numbers of items in a set and found no issues.

**Review by lbiedinger — APPROVED** [review]

File: lang/en.js [hunk 2]
```diff
@@ -475,6 +475,7 @@ export default {
   },
   "items": {
     "itemCount": "1 item |{count} items",
+    "itemOf": "100 of {count} items",
```
Comment (rwd) [inline lang/en.js:478]: 100 should be a variable inserted by interpolation, not hard-coded here.

**Review by rwd — CHANGES_REQUESTED** [review]

File: lang/en.js (hunk 2 above)
Comment (alansutherland) [reply lang/en.js]: ok 100 has been moved to a variable

**Review by rwd — APPROVED** [review]

**Pull request merged**